docs/src/man/basics.md

```diff
@@ -1565,40 +1565,30 @@ julia> german[Not(5), r"S"]
                 984 rows omitted
 ```
 
-## Basic Usage of Transformation Functions
+## Basic Usage of Manipulation Functions
 
-In DataFrames.jl we have five functions that we can be used to perform
-transformations of columns of a data frame:
+In DataFrames.jl there are seven functions
+which can be used to perform operations on data frame columns:
 
-- `combine`: creates a new data frame populated with columns that are results of
-  transformation applied to the source data frame columns, potentially combining
+- `combine`: creates a new data frame populated with columns that result from
+  operations applied to the source data frame columns, potentially combining
   its rows;
 - `select`: creates a new data frame that has the same number of rows as the
-  source data frame populated with columns that are results of transformations
+  source data frame populated with columns that result from operations
   applied to the source data frame columns;
 - `select!`: the same as `select` but updates the passed data frame in place;
 - `transform`: the same as `select` but keeps the columns that were already
   present in the data frame (note though that these columns can be potentially
   modified by the transformation passed to `transform`);
 - `transform!`: the same as `transform` but updates the passed data frame in
   place.
+- `subset`: creates a new data frame populated with the same columns
+as the source data frame, but with only the rows where the passed operations are true;
+- `subset!`: the same as `subset` but updates the passed data frame in place;
 
-The fundamental ways to specify a transformation are:
-
-- `source_column => transformation => target_column_name`; In this scenario the
-  `source_column` is passed as an argument to `transformation` function and
-  stored in `target_column_name` column.
-- `source_column => transformation`; In this scenario we apply the
-  transformation function to `source_column` and the target column names is
-  automatically generated.
-- `source_column => target_column_name` renames the `source_column` to
-  `target_column_name`.
-- `source_column` just keep the source column as is in the result without any
-  transformation;
-
-These rules are typically called transformation mini-language.
-
-Let us move to the examples of application of these rules
+These functions and their methods are explained in more detail in the section
```

Comment:
Maybe add that not only more detail, but with more "slow paced" and informal approach :).